.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,80 @@
+name: Continuous Integration
+
+on: push
```

Comment:
I would make this more specific or else will run for tags and run for every branch.

This is what we used for `node-file-trace` which is master branch and PRs:

```yml
on:
  push:
    branches:
    - master
    tags:
    - '!*'
  pull_request:
```

https://github.com/zeit/node-file-trace/blob/master/.github/workflows/ci.yml